### PR TITLE
Vehicle details

### PR DESCRIPTION
**Pull request by peter-budo**
@karllindmark vehicle details

Main features:
 • Vehicle details layout
 • Added missing resources for expansions

There is small task to finish and that is fetch new image for transport heli z-9 haitun as current image has that grey framing of broken PNG

### CODE REVIEW

**Comment by peter-budo** [comment]
I need to check this on small resolution, landscape of dialog on 4" maybe issue here

File: BF4Intel/src/main/java/com/ninetwozero/bf4intel/ui/stats/vehicles/VehicleDetailFragment.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        Collections.sort(vehicleStats);
+    }
+
+    private void setupTitle() {
+        final String title = getString(VehiclesGroupStringMap.get(groupedVehicles.getGroupName()));
+        setTitle(title);
+    }
+
+    private void populateViews(final View view) {
+
```
Comment (karllindmark) [inline BF4Intel/src/main/java/com/ninetwozero/bf4intel/ui/stats/vehicles/VehicleDetailFragment.java]: Possible empty line (or mess-up by Github).

File: BF4Intel/src/main/res/layout/fragment_vehicle_details.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  </LinearLayout>
+
+  <LinearLayout
+    style="@style/NonListViewCard"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:gravity="center_horizontal"
+    android:divider="@drawable/divider"
+    android:showDividers="middle"
```
Comment (karllindmark) [inline BF4Intel/src/main/res/layout/fragment_vehicle_details.xml:56]: Nice touch, didn't know of it. :-)) :+1:

**Comment by karllindmark** [comment]
@peter-budo: Don't forget the changelog. :-)

**Comment by karllindmark** [comment]
TODO:
- [x] Changelog
- [x] 4" low resolution check (?)
- [ ] Update haitun image (can't do it. We actually discussed this in past when you resized images that this heli is messed up and has some rectangular background)

@peter-budo 
EDIT: last bull point - comment

**Comment by peter-budo** [comment]
Will do

**Comment by peter-budo** [comment]
Here is screenshot from HTC One S, screen size 4.3". I was able scroll that 3 visible options no problems

htc-one-s-landscape-screen-4 3

**Comment by karllindmark** [comment]
@peter-budo: Oh, I thought you meant that the Haitun image was updated on Battlelog, and ready for download. :-)

**Pull request merged**